### PR TITLE
i/9781: Cleaning up the naming convention in README, LICENSE, package.json files

### LINKED ISSUE #9781: Cleaning up the naming convention throughout the packages
## Provide a description of the task

There are some discrepancies when it comes to the naming convention used in some of the common files for the packages, like `README`, `LICENSE`, `package.json`.  
   
package.json examples:  
[Special-characters](https://github.com/ckeditor/ckeditor5/blob/523f88cfc55a920d931c784ab3469e94f749557e/packages/ckeditor5-special-characters/package.json#L4) ->  `"description": "Special characters feature for CKEditor 5.",`   
[Media-embed](https://github.com/ckeditor/ckeditor5/blob/523f88cfc55a920d931c784ab3469e94f749557e/packages/ckeditor5-media-embed/package.json#L4) ->  `"description": "Media Embed feature for CKEditor 5.",`   
[Upload](https://github.com/ckeditor/ckeditor5/blob/523f88cfc55a920d931c784ab3469e94f749557e/packages/ckeditor5-upload/package.json#L4) ->  `"description": "Upload Feature for CKEditor 5.",`   
  
LICENSE.md examples:  
[Restricted editing](https://github.com/ckeditor/ckeditor5/blob/523f88cfc55a920d931c784ab3469e94f749557e/packages/ckeditor5-restricted-editing/LICENSE.md) \->  `**CKEditor 5 restricted editing feature**`   
[Alignment](https://github.com/ckeditor/ckeditor5/blob/523f88cfc55a920d931c784ab3469e94f749557e/packages/ckeditor5-alignment/LICENSE.md) ->  `**CKEditor 5 Text Alignment Feature**`   
  
README.md examples:  
[Enter](https://github.com/ckeditor/ckeditor5/blob/523f88cfc55a920d931c784ab3469e94f749557e/packages/ckeditor5-enter/README.md) ->  `CKEditor 5 Enter feature`   
[Media-embed](https://github.com/ckeditor/ckeditor5/blob/523f88cfc55a920d931c784ab3469e94f749557e/packages/ckeditor5-media-embed/README.md) ->  `CKEditor 5 media embed feature`   
 

## 📃 Other details

*   Browser: …
*   OS: …
*   CKEditor version: …
*   Installed CKEditor plugins: …

### LINKED ISSUE #10045: Inconsistent demo section in our package's README.md
## 📝 Provide a description of requested docs changes

Some feature have mentioned section:

*   [https://github.com/ckeditor/ckeditor5/blob/3e63cf3681ada2eef9fa548a50436658fb8ceabd/packages/ckeditor5-image/README.md#L12](https://github.com/ckeditor/ckeditor5/blob/3e63cf3681ada2eef9fa548a50436658fb8ceabd/packages/ckeditor5-image/README.md#L12)
*   [https://github.com/ckeditor/ckeditor5/blob/3e63cf3681ada2eef9fa548a50436658fb8ceabd/packages/ckeditor5-mention/README.md#L12](https://github.com/ckeditor/ckeditor5/blob/3e63cf3681ada2eef9fa548a50436658fb8ceabd/packages/ckeditor5-mention/README.md#L12)
*   [https://github.com/ckeditor/ckeditor5/blob/3e63cf3681ada2eef9fa548a50436658fb8ceabd/packages/ckeditor5-page-break/README.md#L12](https://github.com/ckeditor/ckeditor5/blob/3e63cf3681ada2eef9fa548a50436658fb8ceabd/packages/ckeditor5-page-break/README.md#L12)

While (vast majority) others don't:

*   [https://github.com/ckeditor/ckeditor5/blob/3e63cf3681ada2eef9fa548a50436658fb8ceabd/packages/ckeditor5-code-block/README.md#L10](https://github.com/ckeditor/ckeditor5/blob/3e63cf3681ada2eef9fa548a50436658fb8ceabd/packages/ckeditor5-code-block/README.md#L10)
*   [https://github.com/ckeditor/ckeditor5/blob/3e63cf3681ada2eef9fa548a50436658fb8ceabd/packages/ckeditor5-restricted-editing/README.md#L10](https://github.com/ckeditor/ckeditor5/blob/3e63cf3681ada2eef9fa548a50436658fb8ceabd/packages/ckeditor5-restricted-editing/README.md#L10)

We should unify this and add demo section to all applicable packages.

### PR DESCRIPTION
**Pull request by mateuszzagorski**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Other: Went through all of the README, LICENSE, package.json files and unified the naming convention used for packages. Closes #9781.

Also closes #10045.

### CODE REVIEW

File: packages/ckeditor5-easy-image/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CKEditor 5 Easy Image with Cloud Services
+CKEditor 5 easy image feature with Cloud Services
```
Comment (psmyrek) [inline packages/ckeditor5-easy-image/README.md:1]: What about other places, where feature name is shown? For example, in this file a few line below there is the following sentence:

> This package implements the [Easy Image](https://ckeditor.com/docs/ckeditor5/latest/features/easy-image.html) feature for CKEditor 5.

Shouldn't that also be changed?

Similar phrases also appear in other files.

Comment (mateuszzagorski) [reply packages/ckeditor5-easy-image/README.md:1]: That's a nice catch, I would think that you're correct. I will review the files again and see what else I'll find.

Comment (godai78) [reply packages/ckeditor5-easy-image/README.md:1]: How about Easy Image being a brand name? This is not just a feature, this is kinda sales point. Like Revision History or smthn?

**Review by godai78 — APPROVED** [review]
This is a massive edit even though easily overlooked changes. Good one.

**Pull request merged**